example/README.md

```diff
@@ -0,0 +1,5 @@
+This example was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Do we need the `example` folder?

Reply:
I'm also seeing  +39,656  changes, are we tracking `node_modules` ?

Reply:
I think we need `example`, I've seen many libraries out there which uses the `example` directory for reference like - 
[react-fontawesome](https://github.com/FortAwesome/react-fontawesome).

Reply:
I saw +39,656 changes too, I think it is because of `package-lock.json` and `yarn-lock`.